components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -171,18 +171,18 @@ export default function () {
             </div>
             <ItemsList className="mt-2">
                 <Item header={true} className="grid grid-cols-3">
-                    <ItemField>
+                    <ItemField className="my-auto">
                         <span>Prebuild</span>
                     </ItemField>
-                    <ItemField>
+                    <ItemField className="my-auto">
                         <span>Commit</span>
                     </ItemField>
-                    <ItemField>
+                    <ItemField className="my-auto">
                         <span>Branch</span>
                     </ItemField>
                 </Item>
                 {prebuilds.filter(filter).sort(prebuildSorter).map((p, index) => <Item key={`prebuild-${p.info.id}`} className="grid grid-cols-3">
-                    <ItemField className="flex items-center">
+                    <ItemField className="flex items-center my-auto">
```

Comment:
Aha, I think here the `my-auto` isn't actually needed (since `flex items-center` already does the vertical centering).

```suggestion
                    <ItemField className="flex items-center">
```

Reply:
Yeah, I just went for a safe refactoring: "monkey pulls `my-auto` out of `ItemField` component, monkey adds `my-auto` to `ItemField` classes everywhere" 😄

Reply:
That's the best kind of refactoring 😁 🙏